rust/src/array.rs

```diff
@@ -554,7 +668,7 @@ impl From<Vec<(Field, ArrayRef)>> for StructArray {
 mod tests {
     use std::thread;
 
-    use super::{Array, BinaryArray, ListArray, PrimitiveArray, StructArray};
+    use super::*;
```

Comment:
Isn't this the one situation where using a wildcard is acceptable (i.e. bringing all the contents of the module into scope so it can be tested)?

Genuinely asking the question on what is best practice rather than disagreeing :)

Reply:
Ah, you're right. It's a very common pattern for tests. Sorry for the noise. 🙂